src/docs/product/performance/metrics.mdx

```diff
@@ -123,16 +123,3 @@ Custom metrics are accessible to you in [sentry.io](https://sentry.io) when you:
 ![Custom metrics in an event detail](custom-metrics-performance.png)
 
 > _Custom metrics data displayed in the sidebar of a transaction event detail under the "Custom Performance Metrics" heading._
-
-## Metrics and Sampling
```

Comment:
in the Dynamic Sampling LA UI (old experience), we still have a link pointing to this section https://github.com/getsentry/sentry/blob/a66d78d30e14f5e86645f1d6a91b296ffb7a09b6/static/app/views/settings/project/server-side-sampling/serverSideSampling.tsx#L509 and by removing it, the link is going to be broken. Do you think we can maybe create a folder like you did for dynamic sampling, so this section will still be available until November 28th?

Reply:
Ah okay. I think I'll have to yeah.